<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckResultWidget</class>
 <widget class="QWidget" name="CheckResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckResultWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="280,102,60,97,20,88,20,88">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QRadioButton" name="radio_auto">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator{image:url(:/img/res/checkwidget/ai.png);}
QRadioButton::indicator:checked{image:url(:/img/res/checkwidget/aip.png);height:32;width:32;}
QRadioButton::indicator:hover{image:url(:/img/res/checkwidget/aih.png);}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QRadioButton" name="radio_manual">
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator{image:url(:/img/res/checkwidget/manual.png);}
QRadioButton::indicator:checked{image:url(:/img/res/checkwidget/manualp.png);}
QRadioButton::indicator:hover{image:url(:/img/res/checkwidget/manualh.png);}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QRadioButton" name="radio_horizontal">
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator{image:url(:/img/res/checkwidget/verti.png);}
QRadioButton::indicator:checked{image:url(:/img/res/checkwidget/vertip.png);height:32;width:32;}
QRadioButton::indicator:hover{image:url(:/img/res/checkwidget/vertih.png);}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QRadioButton" name="radio_vertical">
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator{image:url(:/img/res/checkwidget/hori.png);}
QRadioButton::indicator:checked{image:url(:/img/res/checkwidget/horip.png);height:32;width:32;}
QRadioButton::indicator:hover{image:url(:/img/res/checkwidget/horih.png);}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #642AB5;border-bottom-left-radius:4px;border-top-left-radius:4px;background:#642AB5;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:url(:/img/res/checkwidget/time.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>97</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>97</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top:1px solid #642AB5;border-bottom:1px solid #642AB5;border-right:1px solid #642AB5;border-left:0px solid #642AB5;border-top-right-radius:4px;border-bottom-right-radius:4px;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="time1">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>思源黑体 CN Bold</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;background:#F0F0F0;color:#FA6400;font-size:28px;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time2">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>思源黑体 CN Bold</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;background:#F0F0F0;color:#FA6400;font-size:28px;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font-size:14px;font-family:PingFang SC;color:#642AB5;border:none;</string>
          </property>
          <property name="text">
           <string>秒</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_3">
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{border:1px solid #642AB5;border-radius:4px;background:color #642AB5 url(:/img/res/checkwidget/refuse.png);background-repeat:no-repeat;background-position:center;}QWidget::hover{border:1px solid #642AB5;border-radius:4px;background:#9254DE url(:/img/res/checkwidget/refuse.png);background-repeat:no-repeat;background-position:center;}QWidget:pressed{border:1px solid #642AB5;border-radius:4px;background:#391085 url(:/img/res/checkwidget/refuse.png);background-repeat:no-repeat;background-position:center;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_4">
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{border:1px solid #642AB5;border-radius:4px;background:color #642AB5 url(:/img/res/checkwidget/check.png);background-repeat:no-repeat;background-position:center;}QWidget::hover{border:1px solid #642AB5;border-radius:4px;background:#9254DE url(:/img/res/checkwidget/check.png);background-repeat:no-repeat;background-position:center;}QWidget:pressed{border:1px solid #642AB5;border-radius:4px;background:#391085 url(:/img/res/checkwidget/check.png);background-repeat:no-repeat;background-position:center;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
